README.md

```diff
@@ -9,6 +9,52 @@ images as well as for information regarding contributing and issues.
 The [Docker Hub page] is generated/maintained over in the
 [docker-library/docs repository], specifically in [the `dart` directory].
 
+## Using this image
+
+We recommend creating small runtime images by leveraging Dart's support for
+ahead-of-time (AOT) [compilation to executables][]. This enables creating small
+runtime images (~10 MB).
+
+The following `Dockerfile` performs two steps:
+
+1. Using the Dart SDK in the `dart:stable` image, compiles your server
+(`bin/server.dart`) to an executable (`server`).
+
+1. Assembles the runtime image by combining the compiled server with the Dart VM
+runtime and it's needed dependencies located in `/runtime/`.
+
+```
+# Use latest stable channel SDK.
```

Comment:
See the wording I used [here](https://github.com/dart-lang/dart-docker/blame/794ca592448d784ef40ad773f3f2b6217720a2b7/README.md#L12) for this. I also should have made it clearer that for production we recommend explicitly specifying the desired version. We should make it clear here or in the bullet above that we support two channels (`dart:<version>` as `dart:stable` or `dart:beta` will pull the latest in that channel). For production, we recommend the `stable` channel and using explicit version tags (generally prefer the most explicit version) to ensure deterministic behavior when rebuilding an image. Ex: `dart:2.12.4`. (Might be too much detail here to note that users should also consider pinning to specific digests in production. Details [here](https://docs.docker.com/engine/reference/commandline/pull/#pull-an-image-by-digest-immutable-identifier)).

Reply:
I see, so there essentially isn't a lock file (similar to `pubspec.lock`) for the image version, so if you build the image again, you might get a new version?

Reply:
Yes. you can republish to the same tag multiple times. In practice, that normally won't happen for explicit tags, but publishers can and do republish to less specific tags all the time to ensure they're the latest (like to `stable` in our case, as well as to `latest`, etc.).

Reply:
Got it, updated